test/models/test_prediction.py

```diff
@@ -56,11 +56,12 @@ def test_should_check_if_batch_frames_equivalent_to_no_of_boxes_if_given(
                               batch, predictions, scores,
                               boxes=boxes))
 
+    @unittest.skip('Depricated should remove')
     def test_should_return_list_of_predictions_for_each_frame_in_batch(self):
-        batch = Batch(frames=[Frame(1, np.ones((1, 1)), None)])
+        batch = Batch(frames=[Frame(1, np.ones((1, 1)), None).asdict()])
         predictions = [['A', 'B']]
         scores = [[1, 1]]
-        expected = [Prediction(batch.frames[0], predictions[0], scores[0])]
+        expected = [Prediction(batch.frames, predictions[0], scores[0])]
```

Comment:
`Prediction` was replaced with a more general class. `Outcome`. This used `pd.DataFrames` again. 

We can remove `Prediction` from the code base.

Reply:
Prediction Removed